modules/viewing-metrics-for-the-single-model-serving-platform.adoc

```diff
@@ -27,13 +27,42 @@ endif::[]
++
+[source,subs="+quotes"]
+----
+sum(increase(vllm:request_success_total{namespace='${namespace}',model_name='${model_name}'}[${rate_interval}]))
+----
+
+.. The following query dispalys the number of successful inference requests over a period of time for a model deployed with the TGIS runtime:
++
+[source,subs="+quotes"]
+----
+sum(increase(tgi_request_success{namespace=${namespace}, pod=~'${model_name}-predictor-.*'}[${rate_interval}]))
+----
+
 endif::[]
 ifdef::cloud-service[]
-. As described in link:https://docs.openshift.com/dedicated/observability/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Dedicated) or link:https://docs.openshift.com/rosa/observability/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Service on AWS), use the web console to run queries for `caikit_*`, `tgi_*` and `ovms_*` model-serving runtime metrics. You can also run queries for `istio_*` metrics that are related to OpenShift Service Mesh.
+. As described in link:https://docs.openshift.com/dedicated/observability/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Dedicated) or link:https://docs.openshift.com/rosa/observability/monitoring/managing-metrics.html#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer^] (Red Hat OpenShift Service on AWS), use the web console to run queries for `caikit_*`, `tgi_*`, `ovms_*` and `vLLM:*` model-serving runtime metrics. You can also run queries for `istio_*` metrics that are related to OpenShift Service Mesh. A few examples are listed below:
+.. The following query displays the number of successful inference requests over a period of time for a model deployed with the vLLM runtime:
++
+[source,subs="+quotes"]
+----
+sum(increase(vllm:request_success_total{namespace='${namespace}',model_name='${model_name}'}[${rate_interval}]))
+----
+
+.. The following query displays the number of successful inference requests over a period of time for a model deployed with the TGIS runtime:
++
+[source,subs="+quotes"]
+----
+sum(increase(tgi_request_success{namespace=${namespace}, pod=~'${model_name}-predictor-.*'}[${rate_interval}]))
+----
+
 endif::[]
 
 [role="_additional-resources"]
 .Additional resources
 * link:https://docs.openvino.ai/2024/ovms_docs_metrics.html#available-metrics-families[OVMS metrics^]
 * link:https://github.com/IBM/text-generation-inference?tab=readme-ov-file#metrics[TGIS metrics^]
+* link:https://docs.vllm.ai/en/v0.3.3/serving/metrics.html[vLLM metrics^]
```

Comment:
I'd set  version to at least 0.4.2, since that's what we're shipping:
```suggestion
* link:https://docs.vllm.ai/en/v0.4.2/serving/metrics.html[vLLM metrics^]
```